inputfiles/addedTypes.json

```diff
@@ -19,6 +19,60 @@
                     ]
                 }
             },
+            "DocumentOrShadowRoot": {
+                "name": "DocumentOrShadowRoot",
+                "exposed": "Window",
+                "methods": {
+                    "method": {
+                        "getSelection": {
+                            "name": "getSelection",
+                            "override-signatures": [
+                                "getSelection(): Selection | null"
+                            ]
+                        },
+                        "elementFromPoint": {
+                            "name": "elementFromPoint",
+                            "override-signatures": [
+                                "elementFromPoint(x: number, y: number): Element | null"
+                            ]
+                        },
+                        "elementsFromPoint": {
+                            "name": "elementsFromPoint",
+                            "override-signatures": [
+                                "elementsFromPoint(x: number, y: number): Element[]"
+                            ]
+                        },
+                        "caretRangeFromPoint": {
+                            "name": "caretRangeFromPoint",
+                            "deprecated": 1,
+                            "override-signatures": [
+                                "caretRangeFromPoint(x: number, y: number): Range"
+                            ]
+                        },
+                        "caretPositionFromPoint": {
+                            "name": "caretPositionFromPoint",
+                            "override-signatures": [
+                                "caretPositionFromPoint(x: number, y: number): CaretPosition | null"
+                            ]
+                        }
+                    }
+                },
+                "properties": {
+                    "property": {
+                        "activeElement": {
+                            "name": "activeElement",
+                            "read-only": 1,
+                            "override-type": "Element | null"
+                        },
+                        "styleSheets": {
+                            "name": "styleSheets",
+                            "read-only": 1,
+                            "override-type": "StyleSheetList"
+                        }
+                    }
+                },
+                "no-interface-object": "1"
```

Comment:
We still have `no-interface-object` 😁

Reply:
This was copied from the other location. I don't know what this does.

Reply:
Looks like it's already set internally for all mixin interfaces. I'll remove it.

Reply:
It tells the code that no global object declarations (`declare var DocumentOrShadowRoot`) should be generated. Interfaces may need it because by default they have global constructor objects, whereas mixins do not.